github/files/drupal7/scripts/rr.php

```diff
@@ -0,0 +1,226 @@
+<?php
+
+/**
+ * Root directory of Drupal installation. Note that you can change this
+ * if you need to. It just has to point to the actual root. This assumes
+ * that the php file is being run in sites/all/modules/registry_rebuild.
+ */
+
+ini_set('memory_limit', -1);
+define('DRUPAL_ROOT', './');
```

Comment:
Well, this file will be stored within docroot directory, so no needs to create a patch from your comment, it won't work @Sanchiz